### PR TITLE
Add bindings for rdflib namespaces. Import DCAM.

### PR DESCRIPTION
**Pull request by edmondchuc**
Fixes https://github.com/RDFLib/rdflib/issues/1470

## Proposed Changes

  - Add bindings for rdflib namespaces.
  - Import `DCAM` to be available in the `namespace` package.

### CODE REVIEW

**Comment by nicholascar** [comment]
Oh dear, this seems to have broken a couple of SPARQL tests!

File: rdflib/namespace/__init__.py [hunk 1]
```diff
@@ -333,11 +333,37 @@ def __init__(self, graph):
...
+        self.bind("sh", SH)
+        self.bind("skos", SKOS)
+        self.bind("sosa", SOSA)
+        self.bind("ssn", SSN)
+        self.bind("time", TIME)
+        self.bind("vann", VANN)
+        self.bind("void", VOID)
         self.bind("xsd", XSD)
 
+        # TODO: XMLNS is not a DefinedNamespace.
```
Comment (edmondchuc) [inline rdflib/namespace/__init__.py:364]: Is there unique properties in the XMLNS? If not, then we can remove the todo comment.

**Comment by edmondchuc** [comment]
I have no idea how adding these additions would break the SPARQL tests. I'm keen to look into it if you have any ideas @nicholascar.

File: rdflib/namespace/__init__.py (hunk 1 above)
Comment (nicholascar) [reply rdflib/namespace/__init__.py:364]: No, there are no properties for it. It's not an RDF namespace as such, just a legacy XML one. I _think_ people need it for RDF/XML but no other RDF formats care about it.

**Comment by nicholascar** [comment]
Perhaps the breaking is due to testing that uses a fixed set of namespace PREFIX statements in the SPARQL?

File: rdflib/namespace/__init__.py (hunk 1 above)
Comment (edmondchuc) [reply rdflib/namespace/__init__.py]: Cool thanks, I removed the todo comment in https://github.com/RDFLib/rdflib/pull/1491/commits/ce9e2cf4003b195f15911f45f28e99c2bc59a302.

**Comment by edmondchuc** [comment]
@nicholascar I fixed the tests. 

Essentially what I did was change the SPARQL service queries for DBPedia's URL from `http` to `https` and now it works. 

```sparql
PREFIX ex:<http://example.org/>
select ?s ?p ?o
where {
     SERVICE <https://DBpedia.org/sparql> {
        VALUES (?s ?p ?o) {(ex:a ex:b 1) (<http://example.org/a> <http://example.org/b> 2)}
    }
} 
LIMIT 2
```

No idea why the `http` version works fine with the basic prefix bindings (`rdf`, `rdfs`, etc.) but did not work for the others 🤷‍♂️.

I think it is fine to make this change to the service query URL since it's a functional test to an external service.

**Pull request merged**